datasets/nasa-power.yaml

```diff
@@ -0,0 +1,128 @@
+Name: NASA Prediction of Worldwide Energy Resources (POWER)
+Description: |
+  NASA's goal in Earth science is to observe, understand, and model the Earth system to discover how it is changing, to better predict change, and to understand the consequences for life on Earth. The Applied Sciences Program serves NASA and Society by expanding and accelerating the realization of societal and economic benefits from Earth science, information, and technology research and development. <br/><br/>
+  The NASA [Prediction Of Worldwide Energy Resources (POWER)](https://power.larc.nasa.gov/) Project, a NASA Applied Sciences program, improves the accessibility and usage NASA Earth Observations (EO) supporting community research in three focus areas: 1) renewable energy development, 2) building energy efficiency, and 3) agroclimatology applications. POWER can help communities be resilient amid observed climate variability through the easy access of solar and meteorological data via a verity of access methods. <br/><br/>
+  The latest POWER version includes hourly-based source Analysis Ready Data (ARD), in addition to enhanced daily, monthly, annual, and climatology ARD. The daily time-series spans 40 years for meteorology available from 1981 and solar-based parameters start in 1984. The hourly source data are from Clouds and the Earth's Radiant Energy System (CERES) and Global Modeling and Assimilation Office (GMAO), spanning 20 years from 2001. The hourly data will provide users the ARD needed to model the energy performance of building systems, providing information directly amenable to decision support tools introducing the industry standard EPW (EnergyPlus Weather file). <br/><br/> 
+  POWER also provides parameters at daily, monthly, annual, and user-defined time periods, spanning from 1984 through to within a week of real time. Additionally, POWER provides are user-defined analytic capabilities, including custom climatologies and climatological-based reports for parameter anomalies, ASHRAE® compatible climate design condition statistics, and building climate zones. <br/><br/>
+  The ARD and climate analytics will be readily accessible through POWER's integrated services suite, including the [Data Access Viewer (DAV)](https://power.larc.nasa.gov/data-access-viewer/). The DAV has recently been improved to incorporate updated parameter groupings, new analytical capabilities, and the new data formats.  POWER also provides a complete [API (Application Programming Interface)](https://power.larc.nasa.gov/api/pages/) that allows uses to obtain all the parameters in the DAV plus additional parameters for larger repetitive orders or from within the user\'s own analytic tools. <br/><br/>
```

Comment:
Not sure if these will cause any problems, but should the html tags at the ends of the lines be removed and could this be formatted to be more human readable?